proof/refine/RISCV64/ADT_H.thy

```diff
@@ -7,9 +7,16 @@
 chapter \<open>Abstract datatype for the executable specification\<close>
 
 theory ADT_H
-  imports Syscall_R
+imports
+  "AInvs.ADT_AI"
```

Comment:
Is the new import necessary for something specific or did this just come in from applying the patch?

Reply:
This can be removed!